### PR TITLE
Carinas Homework

### PR DESCRIPTION
**Pull request by carinasauter**

### CODE REVIEW

**Comment by carinasauter** [comment]
my homework

File: HW05_ex00_logics.py [hunk 1]
```diff
@@ -4,38 +4,57 @@
...
-            accepts only non-word numerals
-            must validate
-        Determines if even or odd
-        Prints determination
-        returns None
-    """
-    pass
+    prompt = "Please provide a number to determine if it is even or odd: \n "
+    user_input = int(input(prompt))
+    try:
```
Comment (danielsgriffin) [inline HW05_ex00_logics.py]: why is this your try block?

File: HW05_ex00_logics.py [hunk 2]
```diff
@@ -4,38 +4,57 @@
...
 def find_average():
     """ Takes numeric input (non-word numerals) from the user, one number
     at a time. Once the user types 'done', returns the average.
     """
-    pass
+    user_input = 0
+    count = 0
+    sum_of_input = 0
+    prompt = "Please provide the next number. Once you want the average, please write 'done'.\n"
+    try:
```
Comment (danielsgriffin) [inline HW05_ex00_logics.py]: reduce the size of your try block

File: HW05_ex00_logics.py [hunk 3]
```diff
@@ -4,38 +4,57 @@
...
+        if user_input % 2 == 0:
+            print('Even')
+        else:
+            print('Odd')
+    except:
+        print('You were just supposed to enter an integer...')
 
 
 def ten_by_ten():
```
Comment (danielsgriffin) [inline HW05_ex00_logics.py]: "returns the average"

File: HW05_ex00_logics.py [hunk 4]
```diff
@@ -4,38 +4,57 @@
...
+        number = ' ' + str(number)
+        print(number, end=" ")
+    print("10")
+    for tens in range(10, 100, 10):
+        for ones in range(1, 11):
+            print(tens + ones, end=' ')
+        print()
 
 
 def find_average():
```
Comment (danielsgriffin) [inline HW05_ex00_logics.py]: "returns the average"

**Pull request closed without merging**